tests/Integration/inc/ThirdParty/Plugins/PageBuilder/Elementor/clearCache.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Plugins\PageBuilder\Elementor;
+
+use Brain\Monkey\Functions;
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+
+/**
+ * @covers WP_Rocket\ThirdParty\Plugins\PageBuilder\Elementor::clear_cache
+ * @group Elementor
+ * @group ThirdParty
+ */
+class Test_ClearCache extends FilesystemTestCase {
+	protected $path_to_test_data = '/inc/ThirdParty/Plugins/PageBuilder/Elementor/clearCache.php';
+
+	public function tearDown() {
+		delete_option( 'elementor_css_print_method' );
+
+		parent::tearDown();
+    }
+
+    public function testShouldDoNothingWhenNotExternal() {
```

Comment:
I think also in here we can simply create a single test and send the files & add_option & do_actions which needs to run.
Should we check first that the files are not null?

I was thinking about something like:
```
public function testShouldCleanCache( $files, $elementor_css_print_method, $actions ) {
foreach ( $files as $file ) {
     $this->assertNull( $this->filesystem->getFile( $file ) );
}

add_option( 'elementor_css_print_method',  $elementor_css_print_method );
foreach ( $actions as $action ) {
     do_action( $action );
}

foreach ( $files as $file ) {
     $this->assertNotNull( $this->filesystem->getFile( $file ) );
}
```

Reply:
Done here https://github.com/wp-media/wp-rocket/pull/2458/commits/623a6650057f0d1d0abe16cb0341f3d3becc1ae0